.gitignore

```diff
@@ -1,3 +1,7 @@
+# Compiled binaries and deployment files 
+/bin/
+/deploy/
```

Comment:
I'd like to ignore csi-attacher binary and other compiled artifacts explicitly in `deploy/`. The directory can have useful files that should not be ignored.